core/src/main/java/org/aerogear/mobile/core/configuration/MobileCoreJsonParser.java

```diff
@@ -19,12 +20,17 @@
  */
 public class MobileCoreJsonParser {
 
-    private final Map<String, ServiceConfiguration> values = new TreeMap<>();
+    private static final Map<String, ServiceConfiguration> values = new TreeMap<>();
+    private static final Map<String, String> hashes = new HashMap<>();
```

Comment:
With this change, this class has no state anymore. Consider removing the constructor and making all methods static.

Reply:
Just getting to you comments now, cheers for the input.